### PR TITLE
Date fns on transactions dateTime

### PR DESCRIPTION
**Pull request by amanalili**
#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [ ] Tested on iOS Tablet
- [ ] Tested on small Android

### CODE REVIEW

File: src/util/utils.js [hunk 1]
```diff
@@ -816,3 +816,11 @@ export function getCryptoAmount(
     throw new Error(error)
   }
 }
+
+export function convertTransactionDate(date: number): { date: string, time: string } {
```
Comment (amanalili) [inline src/util/utils.js:820]: export function unixToLocaleDateTime

File: src/util/utils.js [hunk 2]
```diff
@@ -816,3 +816,11 @@ export function getCryptoAmount(
...
+  const txDate = new Date(date * 1000)
```
Comment (amanalili) [inline src/util/utils.js:821]: date

File: src/util/utils.js (hunk 1 above)
Comment (amanalili) [reply src/util/utils.js:820]: date:number to unixDate: number

File: src/actions/TransactionListActions.js [hunk 3]
```diff
@@ -91,18 +91,15 @@ const getAndMergeTransactions = async (state: RootState, dispatch: Dispatch, wal
...
       if (!transactionIdMap[tx.txid]) {
         // if the transaction is not already in the list
         transactionIdMap[tx.txid] = key
         // $FlowFixMe
         transactionsWithKeys.push({
           // then add it
           ...tx,
-          dateString,
-          time,
+          dateString: convertTransactionDate(tx.date).date,
```
Comment (amanalili) [inline src/actions/TransactionListActions.js:101]: const { date, time } = convertTransactionDate(tx.date)

**Review by thehobbit85 — APPROVED** [review]

**Pull request merged**